content/v35/3_date/page.txt

```diff
@@ -7,5 +7,139 @@ Text:
 
 (screenshot: date.png)
 
-Docs for new options are coming soon …
+Gone are the days of a date field with three select inputs with a fixed order. The new fields consist of a single text input with some sugar on tops. Which allows for a range of new possibilities - such as custom display orders and formats - but also provides a much better interaction with e.g. copying and pasting dates/times between fields.
 
+## Display in your preferred format
+
+The fields now support a new `display` option which defines how the date is displayed in the Panel. Offering a range of options:
+
+```yaml
+# default
+display: YYYY-MM-DD
+
+# For ze Germäns
+display: DD.MM.YYYY
+
+# The Brtits
+display: MM/DD/YYYY
+```
+
+The following tokens are supported:
+
+| Token | Output | Description |
+|--|--|--|
+| `YY` | 18 | Two-digit year
+| `YYYY` | 2018 | Four-digit year
+| `M` | 1-12 | Month
+| `MM` | 01-12 | Month, 2-digits
+| `MMM` | Jan-Dec | Abbreviated month name
+| `MMMM` | January-December | Full month name
+| `D` | 1-31 | Day of the month
+| `DD` | 01-31 | Day of the month, 2-digits
+| `H` | 0-23 | Hour
+| `HH` | 00-23 | Hour, 2-digits
+| `h` | 1-12 | Hour, 12-hour clock
+| `hh` | 01-12 | Hour, 12-hour clock, 2-digits
+| `m` | 0-59 | Minute
+| `mm` | 00-59 | Minute, 2-digits
+| `s` | 0-59 | Second
+| `ss` | 00-59 | Second, 2-digits
+| `A` | AM PM	| |
+| `a` | am pm | |
+
+## Parsing your input
+
+Since the field now offers free input, we need to parse that input and turn it into a valid date. To be smart and resourceful about this (instead of bundling huge natural language libraries), the field makes the assumption that the input can be expected to follow some style of the chosen display format. What does this mean? Let's start with the following format
+
+```yaml
+display: DD.MM.YYYY
+```
+
+The field expects any input to match this format as well and converts it to a valid date internally. 
+
+```yaml
+# input -> converted
+29.01.1989 -> 29.01.1989
```

Comment:
I might be blind, but are the two different?

Reply:
No, the point was: if you enter it fully of course it gets parsed correctly but also with the following lines if you enter only partially

Reply:
Ah, ok. I completely misunderstood this.

Reply:
Maybe you have an idea how to make this clearer?

Reply:
I think it's fine. It was me :)